modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListDisplayContext.java

```diff
@@ -148,6 +148,19 @@ public List<CommerceLowStockActivity> getCommerceLowStockActivities() {
 
 	public Map<String, Object> getContext() {
 		return HashMapBuilder.<String, Object>put(
+			"addCPConfigurationEntryRenderURL",
+			() -> PortletURLBuilder.createRenderURL(
+				liferayPortletResponse
+			).setMVCRenderCommandName(
+				"/cp_configuration_lists/add_cp_configuration_entry"
+			).setBackURL(
+				cpRequestHelper.getCurrentURL()
+			).setParameter(
+				"cpConfigurationListId", getCPConfigurationListId()
```

Comment:
question: why here it's just `getCPConfigurationListId()` and on line 175 it's a block of code? Is there any difference between the two?